bentoml/_internal/io_descriptors/json.py

```diff
@@ -42,7 +41,7 @@ def default(self, o: _SerializableObj) -> t.Any:
         if isinstance(o, np.ndarray):
             return o.tolist()
 
-        if any(isinstance(o, i) for i in [pd.DataFrame, pd.Series]):
+        if isinstance(o, (pd.DataFrame, pd.Series)):
             raise BentoMLException(
```

Comment:
Just noticed this exception -why did we make this change? why not just try converting it to JSON here? it is quite common to return a dictionary that contains dataframe and it would be nice to just support that - PandasDataFrame doesn't really handle that case neither

Reply:
how about
```
@api(output=Multipart(df=DataFrame()))
def api():
    return {"df": df}

```

Reply:
shall we encourage this usage since we already supported

```
@api(output=Multipart(df=DataFrame()))
def api():
    return {"df": df}

```
?

Reply:
it's quite different, right? one is a multi-part response vs. a simple JSON response